core/src/main/java/org/apache/calcite/sql/fun/SqlLibrary.java

```diff
@@ -50,7 +50,13 @@
   ORACLE("o"),
   /** A collection of operators that are in PostgreSQL but not in standard
    * SQL. */
-  POSTGRESQL("p");
+  POSTGRESQL("p"),
+  /** A collection of operators that are in BigQuery but not in standard SQL. */
+  BIGQUERY("b"),
+  /** A collection of operators that are in Hive but not in standard SQL. */
+  HIVE("h"),
+  /** A collection of operators that are in Spark but not in standard SQL. */
+  SPARK("s");
```

Comment:
Suggest to use `MySQL` dialect instead of adding new library type just for one function.